pages/a/decred-101.vue

```diff
@@ -43,7 +43,7 @@
             <el-col :lg="16">
               <h1 class="article--title">Decred 101 - Start here</h1>
 
-              <h2>Decred is Sound Money</h2>
+              <h2>Decred is Sound Money - This is El Captain edit</h2>
```

Comment:
Hi @DCR-El-Capitan. This is nice, but we shouldn't edit titles like this.
---- THIS IS FOR TESTING ----